doc/build_apps/js_app_bundle.rst

```diff
@@ -134,25 +134,196 @@ It validates the request body and returns the result of a mathematical operation
 Deployment
 ----------
 
-An app bundle can be wrapped into a governance proposal with the Python client for deployment:
+An app bundle must be wrapped into a JSON object for submission as a ``set_js_app`` proposal to deploy the application code onto a CCF service.
+For instance a proposal which deploys the example app above would look like:
 
-.. code-block:: bash
+.. code-block:: json
+
+    {
```

Comment:
This block is huge, and largely a repetition of the `app.json` we already paste above. But it's a near-minimal example, and I think it's better to be verbose than to trim some sections?